pkg/core/network.go

```diff
@@ -469,12 +469,37 @@ func (n *NetworkCommand) PartitionChain(ipset string) ([]*pb.Chain, error) {
 		return nil, nil
 	}
 
+	chains := make([]*pb.Chain, 0, 2)
+	var toChains, fromChains []*pb.Chain
+	var err error
+
+	if n.Direction == "to" || n.Direction == "both" {
+		toChains, err = n.getPartitionChain(ipset, "to")
+		if err != nil {
+			return nil, errors.WithStack(err)
```

Comment:
If the error is returned by function inside the Chaos Mesh, we could assume this error is already equipped with a stack backtrace, so there is no need to call `errors.WithStack`.

Reply:
The function `getPartitionChain` only returns an error when `direction` is neither `to` nor `from`,   the error is not from Chaos Mesh

Reply:
The error of `getPartitionChain` was generated by `errors.New`, which has equipped with a stack backtrace. So we don't need to use 'errors.WithStack` here.